simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java

```diff
@@ -433,6 +434,10 @@ private void makeInitialReadiumRequest(
     this.readium_js_api = ReaderReadiumJavaScriptAPI.newAPI(in_webview);
     this.simplified_js_api = ReaderSimplifiedJavaScriptAPI.newAPI(in_webview);
 
+    final ReaderSimplifiedJavaScriptAPIType simplified_js =
+      Objects.requireNonNull(this.simplified_js_api);
+    this.simplified_js = simplified_js;
+
```

Comment:
Not too sure about this. Why is the separate but referentially equal `simplified_js` field necessary? The `simplified_js_api` field is assigned once in the activity's `onCreate` method and never reassigned (it would actually be declared `final` if Android allowed for that).

Reply:
I wasn't sure about the syntax or if `Objects.requiredNonNull` was needed. So I can probably just use `this.simplified_js_api` for the calls? I'll try it.

Reply:
> So I can probably just use this.simplified_js_api for the calls? I'll try it.

Yep, you should be able to.

Reply:
Thanks for the explanation. I need to get more acquainted with Java and now Kotlin!